BUILDGUIDE.md

```diff
@@ -173,6 +173,7 @@ Manual Tests require the below setup to run:
   |AzureKeyVaultClientSecret | (Optional) "Client Secret" of the Active Directory registered application, granted access to the Azure Key Vault specified in `AZURE_KEY_VAULT_URL` | _{Client Application Secret}_ |
   |LocalDbAppName | (Optional) If Local Db Testing is supported, this property configures the name of Local DB App instance available in client environment. Empty string value disables Local Db testing. | Name of Local Db App to connect to.|
   |LocalDbSharedInstanceName | (Optional) If LocalDB testing is supported and the instance is shared, this property configures the name of the shared instance of LocalDB to connect to. | Name of shared instance of LocalDB. |
+  |LocalDbNamedPipesEnabled | (Optional) If Local Db Testing is supported, this property enables the run of Local db named pipe instance tests, LocalDbAppName should also ber set | `true` OR `false`|
```

Comment:
This property is redundant. When a Localdb is running, the **Instance pipe name** will be available.

Reply:
I remove this configuration and added the Active issue as discussed.